src/Engine/ProtoCore/ExtendedLibraries/FunctionObject.ds

```diff
@@ -1,3 +1,5 @@
+
+[InternalClass]
```

Comment:
Great, I didn't know that this syntax was supported. Can we have a similar attribute on function definition, to hide the internal methods from the library view. Currently, any method starting with underscore (_) is treated internal.